src/utils.ts

```diff
@@ -197,3 +199,22 @@ export async function getCurrentFileSelectionPath(): Promise<vscode.Uri> {
 export function setContext(inContext: vscode.ExtensionContext) {
 	context = inContext;
 }
+
+export function getLastColumnUsedSetting() : number {
+	let lastColumn : number | undefined = context.workspaceState.get(DIDACT_COLUMN_SETTING);
+	if (!lastColumn) {
```

Comment:
thats a falsy statement, no? I am not sure what is the result if lastColumn is 0 or 1 (true / false). Maybe better check directly for undefined in this case.

Reply:
!lastColumn is false or undefined for sure - I can check for > 0 maybe?

Reply:
No, I mean that the if clause is not just checking for undefined here. If 0 or 1 is returned then it will most likely be interpreted as a boolean. I would do if (lastColumn === undefined) instead.

Reply:
Fixed